tests/Custom/CapabilityTest.php

```diff
@@ -80,18 +86,20 @@ public static function capabilitiesDataProvider(): iterable
     }
 
     /**
-     * @param string $browserName
-     * @param array<string, mixed> $desiredCapabilities
-     * @return WebdriverClassicDriver&object{capabilities: array<string, mixed>}
+     * @param TCapabilities $desiredCapabilities
+     * @return WebdriverClassicDriver&object{capabilities: TCapabilities}
      */
-    private function createDriverExposingCapabilities(string $browserName, array $desiredCapabilities = []): WebdriverClassicDriver
+    private function createDriverExposingCapabilities(string $browserName, array $desiredCapabilities): WebdriverClassicDriver
     {
         return new class($browserName, $desiredCapabilities) extends WebdriverClassicDriver {
             /**
              * @var array<string, mixed>
              */
             public array $capabilities;
 
+            /**
+             * @param array<string, mixed> $desiredCapabilities
```

Comment:
Note that we cannot use TCapabilities here, or on line 96 at the moment, because of https://github.com/phpstan/phpstan/issues/9346.

Reply:
Maybe you can create a regular (not anonymous) class and just auto-load it in the test.

Reply:
At this point it's causing more trouble than being convenient, good point.